src/common/components/first-time-heading/index.js

```diff
@@ -59,11 +59,12 @@ class FirstTimeHeading extends Component {
 
   // TODO: bartaz display state properly (not based on message)
   renderProgress(progress) {
+    progress = [SIGNALS.DONE, SIGNALS.ACTIVE, SIGNALS.DEFAULT];
```

Comment:
is this a test line? it means not being able to set `progress` with `renderProgress`